modules/era/lua/dynamis/mob_spawning_files/dynamis_bastok_mobs.lua

```diff
@@ -0,0 +1,812 @@
+xi.dynamis.mobList[zoneID][121].pos = { 31.6634, 0.0000, -64.0405, 86  } -- W3 Alley W
+xi.dynamis.mobList[zoneID][122].pos = { 39.7658, 0.0000, -59.6651, 64  } -- W3 Alley E
+xi.dynamis.mobList[zoneID][123].pos = { 54.8432, 0.0000, -67.1650, 93  } -- W3 Outside Mog NW
+xi.dynamis.mobList[zoneID][124].pos = { 64.4780, 0.0000, -66.9347, 103 } -- W3 Outside Mog N
+xi.dynamis.mobList[zoneID][125].pos = { 76.5238, 0.0000, -67.1802, 92  } -- W3 Outside Mog NE
+xi.dynamis.mobList[zoneID][126].pos = { 62.2124, 0.0000, -85.9488, 159 } -- W3 Outside Mog SW
+xi.dynamis.mobList[zoneID][127].pos = { 67.9729, 0.0000, -85.3997, 171 } -- W3 Outside Mog S
+xi.dynamis.mobList[zoneID][128].pos = { 77.0071, 0.0000, -79.6646, 140 } -- W3 Outside Mog SE
+xi.dynamis.mobList[zoneID][129].pos = { 90.1017, 0.6234, -71.7804, 128 } -- W3 Mog Clockwise #1
+xi.dynamis.mobList[zoneID][130].pos = { 90.4399, 0.9944, -57.0740, 66  } -- W3 Mog Clockwise #2
+xi.dynamis.mobList[zoneID][131].pos = { 90.7621, 0.9944, -87.7146, 190 } -- W3 Mog Clockwise #12
+xi.dynamis.mobList[zoneID][132].pos = {102.6341, 0.9944, -56.2798, 114 } -- W3 Mog Clockwise #3
+xi.dynamis.mobList[zoneID][133].pos = {102.8179, 0.9944, -88.3096, 146 } -- W3 Mog Clockwise #11
+xi.dynamis.mobList[zoneID][134].pos = {110.9067, 0.9944, -56.7192, 121 } -- W3 Mog Clockwise #4
+xi.dynamis.mobList[zoneID][135].pos = {110.8669, 0.9944, -88.1171, 135 } -- W3 Mog Clockwise #10
+xi.dynamis.mobList[zoneID][136].pos = {101.4331, 0.9944, -65.9948, 185 } -- W3 Mog N
+xi.dynamis.mobList[zoneID][137].pos = {101.6548, 0.9944, -78.9599, 69  } -- W3 Mog S
+xi.dynamis.mobList[zoneID][138].pos = {119.3663, 0.9944, -54.6408, 89  } -- W3 Mog Clockwise #5
+xi.dynamis.mobList[zoneID][139].pos = {118.7241, 0.9944, -88.4772, 161 } -- W3 Mog Clockwise #9
+xi.dynamis.mobList[zoneID][140].pos = {117.2807, 0.9944, -65.8843, 128 } -- W3 Mog Clockwise #6
+xi.dynamis.mobList[zoneID][141].pos = {117.2807, 0.9944, -77.7863, 128 } -- W3 Mog Clockwise #8
+xi.dynamis.mobList[zoneID][142].pos = {117.2442, 0.9944, -71.5537, 128 } -- W3 Mog Clockwise #7
+
+----------------------------------------------------------------------------------------------------
+--                                    Setup of Mob Functions                                      --
+----------------------------------------------------------------------------------------------------
+------------------------------------------
+--             Patrol Paths             --
+------------------------------------------
+-- xi.dynamis.mobList[zoneID][MobIndex].patrolPath = {xpos1,ypos1,zpos1, xpos2,ypos2,zpos2,  xpos3,ypos3,zpos3}
+
+-- xi.dynamis.mobList[zoneID][8  ].patrolPath = { -96,  -2, -123,     -60,  -2, -113,    -96,   -2, -123  }    -- Entrance Bridge W
+
+xi.dynamis.mobList[zoneID].patrolPaths = {6, 35, 11, 21, 14, 12, 9, 34, 25, 32, 58, 60, 59, 61, 63, 62, 75, 76, 70, 71, 47, 49, 50, 104, 102}
```

Comment:
Zach says:

We will need to make more specific paths for mobs that path up and down objects in the Y. (This will fix the statues flying.)

IE many more points in their path.

Reply:
pathing between two points is dangerous if Y is involved. (We would need to look into how onPath works to see if it can be sorted there. But he suggested making pathing more stringent.)